test/quantization/test_quantize_script.py

```diff
@@ -1405,6 +1405,42 @@ def forward(self, x):
         FileCheck().check_not("aten::batch_norm") \
                    .run(model.graph)
 
+    def test_qbatch_norm_relu(self):
+        class M(torch.nn.Module):
+            def __init__(self, inplace):
+                super(M, self).__init__()
+                self.bn = torch.nn.BatchNorm2d(3).to('cpu', torch.float)
+                self.relu = torch.nn.ReLU(inplace=inplace)
+
+            def forward(self, x):
+                return self.relu(self.bn(x))
+
+        class Func(torch.nn.Module):
+            def __init__(self):
+                super(Func, self).__init__()
+                self.bn = torch.nn.BatchNorm2d(3).to('cpu', torch.float)
+
+            def forward(self, x):
+                return F.relu(self.bn(x))
```

Comment:
can you add a `inplace` argument here as well?

Reply:
Added a comment explaining that it is not supported.

Reply:
you can still test for the case the `inplace` argument is a constant I think